app/Http/Backend/Controllers/CategoryController.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+namespace App\Http\Backend\Controllers;
+
+use Illuminate\Http\Request;
+
+use App\Http\Requests;
+use App\Http\Backend\Requests\AddCategoryRequest;
+use App\Http\Backend\Requests\EditCategoryRequest;
+use App\Category;
+
+class CategoryController extends Controller
+{
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        $data = Category::select('id', 'name', 'image')->orderBy('id', 'DESC')->get();
+        return view('backend.category.list', compact('data'));
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('backend.category.add');
+    }
+
+    /**
+     * Store a newly created resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request request
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function store(CategoryRequest $request)
+    {
+        $img = $request->file('image');
+        $imgName = time().'-'.$img->getClientOriginalName();
+        $cate = new Category;
+        $cate->name = $request->txtName;
```

Comment:
`$cate = new Category;` I think this line is not good.
`$cate = new Category($request->all());` is better, then you don't need to assign parameter one by one.